src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ReadMetadata.java

```diff
@@ -71,17 +75,27 @@ public ReadMetadata( final Set<Integer> crossContigIgnoreSet,
                     firstContigID==null ? PartitionBounds.UNMAPPED : firstContigID,
                     stats.getFirstLocation(),
                     lastContigID==null ? PartitionBounds.UNMAPPED : lastContigID,
-                    stats.getLastLocation());
+                    stats.getLastLocation(),
+                    stats.getSpan() );
         }
         final Map<String, LibraryRawStatistics> combinedMaps =
                 perPartitionStatistics.stream()
                         .map(PartitionStatistics::getLibraryNameToStatisticsMap)
                         .reduce(new HashMap<>(), ReadMetadata::combineMaps);
         nReads = combinedMaps.values().stream().mapToLong(LibraryRawStatistics::getNReads).sum();
         final long nReadBases = combinedMaps.values().stream().mapToLong(LibraryRawStatistics::getNBases).sum();
+        avgReadLen = (int)(nReadBases / nReads);
         nRefBases = header.getSequenceDictionary().getSequences()
                 .stream().mapToLong(SAMSequenceRecord::getSequenceLength).sum();
-        coverage = (int)((nReadBases + nRefBases/2) / nRefBases); // rounding the coverage number
+        final float cov = (float)nReadBases / nRefBases;
+        if ( cov >= MIN_COVERAGE ) coverage = cov;
+        else {
+            logger.warn("Apparent coverage (" + cov + ") too low.  Pretending it's 10x.");
```

Comment:
It's true we don't have a great place to put this sort of documentation yet. I guess I was thinking just the javadoc for `FindBreakpointEvidenceSpark` and `StructuralVariationDiscoveryPipelineSpark` for now, just so that it's somewhere.